codemeta.json

```diff
@@ -3,7 +3,6 @@
     "@type": "SoftwareSourceCode",
     "license": "https://spdx.org/licenses/Apache-2.0",
     "codeRepository": "git+https://github.com/o2r-project/erc-checker.git",
```

Comment:
Can you add a link to the GH action here instead?

Reply:
This is now linked to the actions site of the project repo.